frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -17,18 +17,31 @@ const StyledIcon = styled('div')(({ theme }) => ({
 }));
 
 const StyledDescription = styled('div')(({ theme }) => ({
-    fontSize: theme.fontSizes.smallBody,
-    fontWeight: theme.fontWeight.medium,
+    fontSize: theme.typography.body2.fontSize,
```

Comment:
Small point, but if the description is a string, then styled description should probably be a `p` instead of a `div`. I can't suggest it because you haven't changed it, but when you're already in this context it makes sense to have a look at that.